packages/serverless/src/awslambda.ts

```diff
@@ -45,6 +45,12 @@ export interface WrapperOptions {
    * @default false
    */
   captureAllSettledReasons: boolean;
+  /**
+   * Automatically trace all handler invocations.
+   * You may want to disable this if you use express within Lambda (use tracingHandler instead).
+   * @default true
+   */
+  startTransaction: boolean;
```

Comment:
I like tracing here as the boolean because it also indicates to the user that tracing (even when performance is not sampled) will not work if set to false.

```suggestion
  startTrace: boolean;
```

Reply:
@AbhiPrasad good point. `startTransaction` is also the name of a function throughout the rest of the codebase, so having it be a boolean here is confusing. Just pushed the change!